src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/transformer/OasysTransformer.kt

```diff
@@ -149,6 +149,20 @@ fun risks(
   alerts = activeAlerts?.map { Alert(it.alertTypeDescription) },
 )
 
+fun populateOSPIScore(oasysOffendingInfo: OasysOffendingInfo?): String? =
+  if (oasysOffendingInfo?.ospIICRisk != null) {
+    oasysOffendingInfo.ospIICRisk
+  } else {
+    oasysOffendingInfo?.ospIRisk
+  }
+
+fun populateOSPCScore(oasysOffendingInfo: OasysOffendingInfo?): String? =
+  if (oasysOffendingInfo?.ospDCRisk != null) {
+    oasysOffendingInfo.ospDCRisk
+  } else {
+    oasysOffendingInfo?.ospCRisk
+  }
+
```

Comment:
`oasysOffendingInfo?.ospDCRisk ?: oasysOffendingInfo?.ospCRisk`

Thoughts?

Reply:
Similar approach for `populateOSPIScore` as well?

Reply:
fair enough  :-)